gitcha_test.go

```diff
@@ -30,6 +31,52 @@ func TestGitRepoForPath(t *testing.T) {
 	}
 }
 
+func TestFindAllFiles(t *testing.T) {
+	tmp := t.TempDir()
+
+	gitignore, err := os.Create(path.Join(tmp, ".gitignore"))
```

Comment:
I think `filepath.Join` is more appropriate here, e.g. for platforms like Windows.

Reply:
Fixed!